packages/console/src/pages/Connectors/components/CreateForm/constants.ts

```diff
@@ -0,0 +1,12 @@
+export const featuredConnectorTargets = [
+  'google',
+  'apple',
+  'facebook',
+  'github',
+  'discord',
+  'wechat',
+  'alipay',
+  'kakao',
+  'naver',
+  'azuread',
+];
```

Comment:
should it be placed somewhere common? Or console is fine?

Reply:
LOG-4865 the "list" should be placed somewhere too